src/ivectorbin/compute-vad-from-frame-likes.cc

```diff
@@ -0,0 +1,186 @@
+// ivectorbin/compute-vad-from-frame-likes.cc
+
+// Copyright  2015 David Snyder
+
+// See ../../COPYING for clarification regarding multiple authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+// WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+// See the Apache 2 License for the specific language governing permissions and
+// limitations under the License.
+
+#include "base/kaldi-common.h"
+#include "util/common-utils.h"
+#include "matrix/kaldi-matrix.h"
+#include "util/stl-utils.h"
+
+using namespace kaldi;
```

Comment:
I think this is disallowed by the Google style guide.  Generally what I do is just put the functions being defined in namespace Kaldi.  Or you could have the using-directive inside the function if really needed.